doc/markdown.md

```diff
@@ -1,24 +1,25 @@
----
-title: Markdown
-description: Markdown
----
-
 # Markdown
 
+The site's Markdown processing uses
+[Kramdown's GitHub Flavored Markdown parser][kramdown-gfm],
+meaning you can write using [GitHub Flavored Markdown][].
+
 ## Code blocks
 
 There are two ways to document code blocks on our sites:
 
  1. Use standard markdown syntax for fenced code like this:
 
-        ```dart
-        class C { ... }
-        ```
+    ````markdown
+    ```dart
+    class C { ... }
+    ```
+    ````
 
  1. Use the [prettify][] Jekyll block tag, supported via a custom plugin
     designed for use on our sites:
```

Comment:
I thought we no longer use or recommend prettify?

Reply:
You still need to use it if you want highlighting on a snippet that does not have a code-excerpt. I've added a note about that until we can migrate completely away from it. Related issue: https://github.com/dart-lang/site-www/issues/3846